minidump-processor/src/process_state.rs

```diff
@@ -416,6 +422,37 @@ impl PossibleBitFlip {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum CrashReasonInconsistency {
+    IntDivByZeroNotPossible,
+    PrivInstructionCrashWithoutPrivInstruction,
+    NonCanonicalAddressFalselyReported,
+    AccessViolationWhenAccessAllowed,
+    CrashingAccessNotFoundInMemoryAccesses,
```

Comment:
We want these to be machine-readable in the JSON output so it's fine for them to be descriptive labels but not quite human-readable phrases. However we use snake-case names for the rest of the JSON output, so it'd be better to print them out that way for consistency.